runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomPlugin.java

```diff
@@ -85,7 +90,7 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 		int[] intStack = client.getIntStack();
 		int intStackSize = client.getIntStackSize();
 
-		if ("scrollWheelZoom".equals(event.getEventName()) && zoomConfig.requireControlDown() && !controlDown)
+		if ("scrollWheelZoom".equals(event.getEventName()) &&  zoomConfig.controlFunction() == ControlFunction.CONTROL_TO_ZOOM && !controlDown)
```

Comment:
Reorder the args to perform the cheaper check(s) first.

```suggestion
		if (!controlDown && "scrollWheelZoom".equals(event.getEventName()) && zoomConfig.controlFunction() == ControlFunction.CONTROL_TO_ZOOM)
```